internal/gatewayserver/database/dbRepo/dbRepo.go

```diff
@@ -0,0 +1,32 @@
+package dbRepo
```

Comment:
As this package doesn't contain application specific logic, would it make sense to move this to `spine` or `spine/db` package?

Reply:
will refactor